pkg/crc/preflight/preflight_linux.go

```diff
@@ -43,7 +43,14 @@ func libvirtPreflightChecks(distro *linux.OsRelease) []Check {
 			checkDescription: "Checking if user is part of libvirt group",
 			check:            checkUserPartOfLibvirtGroup,
 			fixDescription:   "Adding user to libvirt group",
-			fix:              fixUserPartOfLibvirtGroup(distro),
+			fix:              fixUserPartOfLibvirtGroup,
+		},
+		{
+			configKeySuffix:  "check-libvirt-group-active",
+			checkDescription: "Checking if active user/process is currently part of the libvirt group",
+			check:            checkCurrentGroups(distro),
+			fixDescription:   "Current user has beed added to the libvirt group, but you need to logout, login again, and run crc setup again for the addition to become effective.",
```

Comment:
@cfergeau this will be displayed even the `checkCurrentGroups` return the error about `User in the currently active process is not part of the libvirt group` which makes this message a bit confusing since as part of check we are saying that user is not part of libvirt group and here it is contradict with that. May be in fixDescription message we drop `Current user has beed added to the libvirt group` ?

Reply:
Still need different team member opinion around this @gbraad @guillaumerose wdyt?

Reply:
Ah sorry, I was so annoyed at my fight with CI/e2e tests that I totally forgot this other comment

Reply:
I changed it now, let me know if it's better.